streamz/tests/test_kafka.py

```diff
@@ -0,0 +1,86 @@
+import os
+from time import sleep
+import requests
+import subprocess
+import pytest
+from contextlib import contextmanager
+from tornado import gen
+
+from ..core import Stream
+from streamz.utils_test import gen_test
+
+KAFKA_FILE = 'kafka_2.11-1.0.0'
+LAUNCH_KAFKA = False
```

Comment:
Perhaps this should be an environment variable - test works fine if kafka happens to be running already. I didn't really want to import the test hooks in confluent here, which are rather more than we need.